autogen/agentchat/conversable_agent.py

```diff
@@ -1469,16 +1477,16 @@ def execute_function(self, func_call, verbose: bool = False) -> Tuple[bool, Dict
         func_name = func_call.get("name", "")
         func = self._function_map.get(func_name, None)
 
+        input_string = self._format_json_str(func_call.get("arguments", "{}"))
```

Comment:
Are the changes in `autogen/agentchat` something that can be upstreamed?

Reply:
It's pretty horrible return code so maybe not this time.